samsungctl/remote_websocket.py

```diff
@@ -0,0 +1,72 @@
+import base64
+import json
+import logging
+import socket
+import time
+import websocket
```

Comment:
I updated the PR with a slightly different approach.

The `Remote` class can now be used just as before. This avoids the obvious breaking change and keeps the public "api" of this module simple for any dependees, for example [home-assistant](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/media_player/samsungtv.py#L64).

I also moved the import to the _RemoteWebsocket_'s `__init__` method. This way, if someone specifies the _websocket_ method, we assume that the dependency was installed.

What do you think?